kubernetes/cray-opa/tests/opa/xname-enabled/run_tests/run_tests.go

```diff
@@ -343,6 +343,7 @@ func main() {
 			"opa": map[string]interface{}{
 				"xnamePolicy": map[string]interface{}{
 					"enabled":   "True",
+          "bos":       "True",
```

Comment:
should run this through the go formatter.

Reply:
Looks like this is a tab vs spaces issue.

Reply:
I've converted all tabs to spaces in all the run_tests.go files to fix these formatting issues.

Reply:
I went back and ran gofmt on it. Go appears to want to have a mix of tabs and spaces for alignment and I'm going to let the tool deal with it.